include/sched.h

```diff
@@ -91,6 +91,13 @@ void sched_clock(void);
  */
 void sched_switch(void);
 
+/*! \brief Switch out to another thread if you shouldn't be running anymore.
+ *
+ * This function will switch if your time slice expired or a thread with higher
```

Comment:
`higher` twice?

Reply:
Perhaps you should add a note here, that this function does not detect if a thread should switch, but rather checks it and executes if needed.

Reply:
Done.